cmd/nomos/vet/vet_impl.go

```diff
@@ -102,9 +102,9 @@ func runVet(ctx context.Context, namespace string, sourceFormat filesystem.Sourc
 		files = filesystem.FilterHierarchyFiles(rootDir, files)
 	case filesystem.SourceFormatUnstructured:
 		if namespace == "" {
-			validateOpts = parse.OptionsForScope(validateOpts, declared.RootReconciler)
+			validateOpts = parse.OptionsForScope(validateOpts, declared.RootReconciler, "")
 		} else {
-			validateOpts = parse.OptionsForScope(validateOpts, declared.Scope(namespace))
+			validateOpts = parse.OptionsForScope(validateOpts, declared.Scope(namespace), "")
```

Comment:
Seems a bit weird passing empty syncName here?

Reply:
moved syncName setter out of `parse.OptionsForScope`.